.github/workflows/ci.yml

```diff
@@ -1,34 +1,39 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Why we need to specify `types`?

Reply:
I seem to recall a time where this was not the default, seems like it is now though. Good catch, reverted.